### ISSUE TITLE
wpt lint only considers <script> elements in the HTML namespace

### ISSUE THREAD

**Issue by fsoder** [web-platform-tests/wpt#11194]
When looking for `testharness.js` / `testharnessreport.js`, the linter only checks for `<script>` elements in the HTML namespace (which has a `src` attribute in the default namespace.) For SVG `<script>` elements, the element namespace will be the SVG namespace, and the attribute will be either `href` (in the default NS) or `href` in the XLink namespace.

**Comment by gsnedders** [comment]
Note this just matches what the manifest does.

**Comment by gsnedders** [comment]
I don't know if we actually want to change this, given there's no clear alternative to `meta` or `link` (for testharness.js options and reftests respectively) so there's going to be plenty of tests that still need to use things in the HTML namespace anyway?

**Comment by fsoder** [comment]
Yes, it's usually not a problem to just use HTML `<script>` (maybe even less typing most of the time =P.) But since using SVG "containers" for these types of tests under `svg/` (rather than, say, HTML) seems to be preferred, I figured I should at least raise this...

**Comment by stephenmcgruer** [comment]
According to @fsoder  in https://chromium-review.googlesource.com/c/chromium/src/+/2042125, this lint check may have broken at some point. They had accidentally exported an SVG test to WPT without putting the testharness.js includes in a `h:` namespace, but nothing caught it.

It's also possible the lint check that was being referenced wasn't expected to catch this, I haven't yet looked into the linter code.
